euth/phases/apps.py

```diff
@@ -1,6 +1,11 @@
 from django.apps import AppConfig
 
+from . import content
 
 class PhasesConfig(AppConfig):
     name = 'euth.phases'
     label = 'euth_phases'
+
+    def ready(self):
+        from django.utils.module_loading import autodiscover_modules
+        autodiscover_modules('phases', register_to=content)
```

Comment:
You can also use `self.module.content` instead of `content` (see [AppConfig.module](https://docs.djangoproject.com/en/1.8/ref/applications/#django.apps.AppConfig.module)).